streamflow/cwl/step.py

```diff
@@ -141,25 +141,27 @@ async def _process_file_token(
 
 
 async def build_token(
-    job: Job,
-    token_value: Any,
     cwl_version: str,
+    inputs: MutableMapping[str, Any],
     streamflow_context: StreamFlowContext,
+    token_value: Any,
+    job: Job | None = None,
```

Comment:
If this is only used to check if files should be processed, it should be better to pass a more significant flag like `process_files: bool = False`, so that the API is clearer